app/src/main/java/org/thunderdog/challegram/ui/ProfileController.java

```diff
@@ -3835,6 +3835,23 @@ private void buildEditCells () {
       items.add(new ListItem(ListItem.TYPE_DESCRIPTION, 0, 0, R.string.AggressiveAntiSpamDesc));
     }
 
+    /*TdApi.SuggestedAction[] actions = tdlib.getSuggestedActions();
+    for (TdApi.SuggestedAction action : actions) {
+      UI.showToast(action.toString(), Toast.LENGTH_SHORT);
+      if (!tdlib.isBroadcastConvertSuggestion(action)) {
+        continue;
+      }
+      if (action.getConstructor() ==  TdApi.SuggestedActionConvertToBroadcastGroup.CONSTRUCTOR) {*/
+        if (supergroup != null && supergroup.memberCount >= 200000 || BuildConfig.DEBUG && tdlib.canConvertToBroadcast(chat.id)) {
```

Comment:
1. Missing check whether `SuggestedActionConvertToBroadcastGroup` is provided by server for current chat
2. `200000` should be replaced with `tdlib.supergroupMaxSize() - 50 /*little margin for leaving/joining members*/`

Reply:
> Missing check whether SuggestedActionConvertToBroadcastGroup is provided by server for current chat

I have no idea how to do it, being that for some strange reason I don't get the notification suggestion to convert the group into a broadcast group